08_model-compression.ipynb

```diff
@@ -634,6 +634,8 @@
     "        self.teacher_model = teacher_model\n",
     "\n",
     "    def compute_loss(self, model, inputs, return_outputs=False):\n",
+    "        device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
+    "        inputs = inputs.to(device)\n",
```

Comment:
The fix is to simply ensure that the inputs are also on the same device

<img width="848" alt="image" src="https://user-images.githubusercontent.com/338917/187307751-9f2103dc-65ef-4f42-9b3f-23c945d930aa.png">